modules/pcd/src/lib/parse-pcd.ts

```diff
@@ -6,11 +6,10 @@
 // @author Filipe Caixeta / http://filipecaixeta.com.br
 // @author Mugen87 / https://github.com/Mugen87
 
-import {MeshAttribute, MeshAttributes} from '@loaders.gl/schema';
-import {getMeshBoundingBox} from '@loaders.gl/schema';
-import {decompressLZF} from './decompress-lzf';
-import {getPCDSchema} from './get-pcd-schema';
-import type {PCDHeader, PCDMesh} from './pcd-types';
+import { getMeshBoundingBox, MeshAttribute, MeshAttributes } from '@loaders.gl/schema';
+import { decompressLZF } from './decompress-lzf';
+import { getPCDSchema } from './get-pcd-schema';
+import type { PCDHeader, PCDMesh } from './pcd-types';
```

Comment:
The spaces you added are causing the lint to fail. Regardless of your personal preferences, when you are contributing to another repo, you need to adopt the conventions in that repo:
```suggestion
import {getMeshBoundingBox, MeshAttribute, MeshAttributes} from '@loaders.gl/schema';
import {decompressLZF} from './decompress-lzf';
import {getPCDSchema} from './get-pcd-schema';
import type {PCDHeader, PCDMesh} from './pcd-types';
```